### PR TITLE
contributing: add page on how to release

### PR DESCRIPTION
**Pull request by julianoes**
I thought as I'm doing the release I should write the steps down.

### CODE REVIEW

**Comment by julianoes** [comment]
@hamishwillee thanks for your changes, however, they should have been an addition for the docs release process. I was describing the code/library release!

**Comment by hamishwillee** [comment]
Bugger. Sorry. Too focussed on my only little bit of the world. Are you OK to under and we can add the docs stuff later?

**Comment by julianoes** [comment]
I'll fix it and add the docs steps, no worries.

File: en/contributing/release.md [hunk 1]
```diff
@@ -5,28 +5,40 @@ These are the instructions on how to get a release out the door.
...
 1. Check if all open pull requests are merged, that need to go in.
-2. Check that CI on `develop` passed.
-3. [Generate the docs](build.md#build_api_reference) and update the reference docs.
-4. Check or update the examples in the docs.
-5. Merge `develop` into `master` and tag the release:
+1. Check that CI on `develop` passed.
+1. [Generate the docs](build.md#build_api_reference) and update the reference docs.
+1. Check or update the examples in the docs.
+1. Run link checker and fix any breaks.
+1. Merge `develop` into `master`:
```
Comment (JonasVautherin) [inline en/contributing/release.md]: Is it normal that it's all `1.`? I guess it is, since it got changed everywhere :thinking:

Comment (julianoes) [reply en/contributing/release.md]: That's a feature of markdown. If you don't want to bother with actually numbering things you can just use `1.`.

**Pull request merged**